scripts/setup-jest.ts

```diff
@@ -73,6 +75,10 @@ const getMockVSCode = () => {
         onDidDelete: jest.fn()
       }),
       workspaceFolders: []
+    },
+    StatusBarAlignment: {
```

Comment:
can we put this in alpha order by property name? So move it to after ProgressLocation